azure_functions_worker/testutils.py

```diff
@@ -963,11 +985,16 @@ def _setup_func_app(app_root):
     extensions = app_root / 'bin'
     ping_func = app_root / 'ping'
     host_json = app_root / 'host.json'
+    extensions_csproj_file = app_root / 'extensions.csproj'
 
     if not os.path.isfile(host_json):
         with open(host_json, 'w') as f:
             f.write(HOST_JSON_TEMPLATE)
 
+    if not os.path.isfile(extensions_csproj_file):
+        with open(extensions_csproj_file, 'w') as f:
+            f.write(EXTENSION_CSPROJ_TEMPLATE)
+
```

Comment:
Writing the new csproj file in the function app for host to pick up the extensions